awx/main/tests/functional/test_feature_flags_api.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
```

Comment:
first line also seems unnecessary

I'd also like to move the file. Elsewhere we have `awx/main/tests/functional/dab_rbac/` type folders. I'd like this to follow that pattern in some form. Even if you created a new `dab_feature_flags/` folder, that would at least be consistent so it'll be much easier to consolidate in the future.